packages/mobx-state-tree/src/types/utility-types/union.ts

```diff
@@ -62,7 +72,11 @@ export class Union extends Type<any, any> {
 
         // find the most accomodating type
         const applicableTypes = this.types.filter(type => type.is(value))
-        if (applicableTypes.length > 1)
+        if (applicableTypes.length > 1) {
+            debugger
```

Comment:
this should probably be removed before release :)

Reply:
Yikes!  Thanks!